app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ErrorsController < ApplicationController
```

Comment:
Normally, you always want to inherit from ApplicationController but this is a special case: let's inherit from `ActionController::Base`. The reason is that if there's a bug actually in ApplicationController, it might not be possible to load error pages if we inherit from it.